client/lcd/swagger-ui/swagger.yaml

```diff
@@ -2238,6 +2238,10 @@ definitions:
         type: array
         items:
           $ref: "#/definitions/Coin"
+      gas_prices:
+        type: array
+        items:
+          $ref: "#/definitions/DecCoins"
```

Comment:
Do we need to add a `#/definitions/DecCoins` in this file as well?

Reply:
Hmmm, I couldn't find any actual "definitions". Will need some help/insight here.

Reply:
Oh! Well in that case! Ignore this plz!